test-status/README.md

```diff
@@ -25,10 +25,10 @@ The App has the following API points, which are to be triggered from Travis CI
 runs. If the `TRAVIS_IP_ADDRESSES` environment variable is set, only requests
 from this comma separated list of IP addresses will be processed.
 
-* `POST /v0/tests/:headSha/:type/:status(queued|started|skipped)`
+* `POST /v0/tests/:headSha/:type/:subType/:status(queued|started|skipped)`
```

Comment:
nit: No colon before `status`, correct?

Reply:
It's how the [URL is defined in express.js](https://expressjs.com/en/guide/routing.html), `status` is the name of the variable, and `(queued|started|skipped)` are its possible values - so a valid URL would be `/v0/tests/abc123/unit/saucelabs/started` where `subType` is assigned the string `'saucelabs'` and `status` is assigned the string `'started'` - the word "status" never appears in the URL

Reply:
Aha, `:status` is a "variable" while `report` in the line below is a "constant". Never mind, carry on.